v2.0/select-clause.md

```diff
@@ -1,52 +1,70 @@
 ---
-title: SELECT
-summary: The SELECT statement retrieves data from a table.
+title: Simple SELECT Clause
+summary: The Simple SELECT clause loads or computes data from various sources.
 toc: false
+redirect-from: select.html
```

Comment:
One more field to add to the front-matter to make our version switcher work between 1.1 and 2.0:

```
key: select.html
```